src/containers/pages/FocusInvestigation/active/tests/__snapshots__/index.test.tsx.snap

```diff
@@ -9536,4 +9536,1648 @@ exports[`containers/pages/ActiveFocusInvestigation renders ActiveFocusInvestigat
 </Router>
 `;
 
-exports[`containers/pages/ActiveFocusInvestigation works with the Redux store 1`] = `null`;
+exports[`containers/pages/ActiveFocusInvestigation works with the Redux store 1`] = `
```

Comment:
we've got smoke without fire over here. 

Could not find a corresponding change in tests that explains this snapshot diff. Which is very wierdly mysterious.